packages/client-core/src/user/components/UserMenu/menus/ProfileMenu.tsx

```diff
@@ -109,14 +109,15 @@ const ProfileMenu = ({ hideLogin, onClose, isPopover }: Props): JSX.Element => {
   const isGuest = selfUser.isGuest.value
   const acceptedTOS = !!selfUser.acceptedTOS.value
 
-  const checkedTOS = useHookstate(acceptedTOS)
+  const checkedTOS = useHookstate(!isGuest)
+  const checked13OrOver = useHookstate(!isGuest)
   const checked18OrOver = useHookstate(acceptedTOS)
-  const hasAcceptedTermsAndAge = checkedTOS.value && checked18OrOver.value
+  const hasAcceptedTermsAndAge = checkedTOS.value && checked13OrOver.value
 
   const originallyAcceptedTOS = useHookstate(acceptedTOS)
 
   useEffect(() => {
-    if (!originallyAcceptedTOS.value && hasAcceptedTermsAndAge) {
+    if (!originallyAcceptedTOS.value && checked18OrOver) {
```

Comment:
This needs to be checking checked18OrOver.value, as otherwise the reference to the hookstate proxy always reads as truthy. Should probably make the useEffect listener also listen on checked18OrOver.value too.

Reply:
This is what I'm currently unsure on. If a user checks 18+ they get access to media/chat, but why would it ever be valid for them to uncheck it? In most flows, you either enter a birthday or do this upon sign up, in which the option is not available once checked. This is kind of how it is implemented as a quick implementation until the new UI & flow is in, just to ensure we have a minimum level of compliance.

Reply:
Oh, and the checkbox is disabled once the user checks it, so they can't uncheck it.

Reply:
Maybe I was thrown off by what the if statement bug was doing. Are users now only given media chat access when they log in and verify they're 18? If so, then this seems fine with the fix.